### PR TITLE
Blog app - Views

### PR DESCRIPTION
**Pull request by muskan2532150**
- Implement the design from the sneak peek wireframes
- [wireframe1](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_root_users.png)
-  [wireframe2](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_user_page.png)
- [wireframe3](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_user_all_posts.png)
- [wireframe4](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_single_post.png)
- Use methods in the [Blog App - processing data in models](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md).

### CODE REVIEW

File: app/views/users/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<section>
+<ul>
+<% @user.each do |users| %>
+<li> <%= render 'user',users:users %> </li>
```
Comment (codecaiine) [inline app/views/users/index.html.erb:4]: - [ ] According to the project requirement when we click on each, it should take us to their show page. Unfortunately, when I clicked in I could not go to the page I had to type the URLs. Please fix it and also make sure that your design matches with the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_page.png) as require for this project project :+1:

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+<div class="bio-container">
+<h3 class="bio">Bio</h3>
+<p><%= @user.bio%></p>
+<%@user.three_recent_posts.each_with_index do |pt,index|%>
+<%= render 'posts/post', post:pt, index:index %>
+<%end%>
+</div>
+<button><%= link_to 'See All Posts', user_posts_path(@user), class: 'btn' %>
+</button>
+</section>
```
Comment (codecaiine) [inline app/views/users/show.html.erb]: - [ ] Good job so far but make sure that your **See all Post** links open user's post index has a design match with the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_all_posts.png). You design is a bit different. Please fix it as expected for this project :+1:

File: app/views/posts/show.html.erb [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+        <%=@post.text%>
+      </div>
+    </div>
+    <div class="comments">
+        <% @post.comments.each do |comment| %>
+          <%= render 'comment', comment: comment %>
+        <% end %>
+    </div>
+ </div>
+ </section>
```
Comment (codecaiine) [inline app/views/posts/show.html.erb:17]: - [ ] Good job so far unfortunately I can't go to the post-show page. Please make sure to have a link takes to the post-show page and make sure that it displays the post with all comments listed as shown on  [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_single_post.png) :+1:

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @muskan2532150 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Good project setup :heavy_check_mark: 
- No rubocop errors :heavy_check_mark: 

## Required Changes ♻️
- [ ] Please make sure that your project design matches the [wireframe required](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#project-requirements) for this project and also make sure that the links and routes taken to the different pages work as expected :+1: 

_Check the comments under the review._

## Optional suggestions
- [ ] [OPTIONAL] The link to your licence is broken , please make sure to fix it to make your readme more professional :+1: 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mwanawabangona — APPROVED** [review]
Hi @muskan2532150,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations!  


## Highlights
*   No linter errors :heavy_check_mark:
*   Repo looks professional :briefcase:
*   Solid pull request scroll
*   Good quality commits
*   Good job implementing the requested changes !! :clap: :clap: :clap:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**